hummingbot/connector/exchange/bloxroute_openbook/bloxroute_openbook_exchange.py

```diff
@@ -60,32 +63,30 @@ class BloxrouteOpenbookExchange(ExchangePyBase):
     def __init__(
         self,
         client_config_map: "ClientConfigAdapter",
-        bloxroute_api_key: str,
+        bloxroute_auth_header: str,
         solana_wallet_public_key: str,
         solana_wallet_private_key: str,
-        open_orders_address: str,
         trading_pairs: Optional[List[str]] = None,
         trading_required: bool = True,
     ):
         """
-        :param bloxroute_api_key: The bloxRoute Labs authorization header to connect with solana trader api
+        :param bloxroute_auth_header: The bloxRoute Labs authorization header to connect with solana trader api
         :param solana_wallet_private_key: The secret key for a solana wallet
         :param trading_pairs: The market trading pairs which to track order book data.
         :param trading_required: Whether actual trading is needed.
         """
 
-        self.logger().exception("creating blox route exchange")
-        self.logger().exception("api key is " + bloxroute_api_key)
-        self.logger().exception("pub key is " + solana_wallet_public_key)
-        self.logger().exception("private key is " + solana_wallet_private_key)
-        self.logger().exception("open orders address is " + open_orders_address)
+        self.logger().exception("Creating bloXroute exchange")
+        self.logger().exception("API Key is " + bloxroute_auth_header)
+        self.logger().exception("Public Key is " + solana_wallet_public_key)
+        self.logger().exception("Private Key is " + solana_wallet_private_key)
```

Comment:
these shouldn't be excpetion level logs, and I think you probably don't want to include these at all. Likely not the best idea to print privat ekey to logs